packages/types/src/lib/phuture-product.ts

```diff
@@ -0,0 +1,22 @@
+import { Anatomy } from './anatomy';
+
+/** ### Phuture Product interface */
+export interface PhutureProduct {
+	/**
+	 * ### Anatomy of Phuture Product
+	 * @returns {Promise<Anatomy>} Anatomy of the product
+	 */
+	anatomy(): Promise<Anatomy>;
+
+	/**
+	 * ### Inactive anatomy of Phuture Product
+	 * @returns {Promise<Anatomy>} Inactive anatomy of the product
+	 */
+	inactiveAnatomy(): Promise<Anatomy>;
```

Comment:
I want to say we should make those methods private on Index class, an don't have them at all on SV

Reply:
So frontend is not invoking `inactiveAnatomy` or `anatomy` functions at all?